cli/testutil/testcli/BUILD

```diff
@@ -7,6 +7,9 @@ go_library(
     data = ["//cli/cmd/bb"],
     importpath = "github.com/buildbuddy-io/buildbuddy/cli/testutil/testcli",
     visibility = ["//visibility:public"],
+    x_defs = {
+        "BBRunfilePath": "$(rlocationpath //cli/cmd/bb)",
```

Comment:
nit: lowercase `BB` to make this private?

`bbRunfilePath`

(here and in the .go file)